examples/run_composer_trainer.py

```diff
@@ -45,7 +45,7 @@ def _main() -> None:
         pass
     else:
         if wandb.run is not None:
-            wandb.config.update(**hparams.to_dict())
+            wandb.config.update(hparams.to_dict())
```

Comment:
Why no more `**`?

Reply:
Unpacking the dictionary didn't work; it threw this exception:

```
Traceback (most recent call last):
  File "/home/ravi/ravi-composer/examples/run_composer_trainer.py", line 70, in <module>
    _main()
  File "/home/ravi/ravi-composer/examples/run_composer_trainer.py", line 48, in _main
    wandb.config.update(**hparams.to_dict())
TypeError: Config.update() got an unexpected keyword argument 'model'
```
The docs here suggest that a dict should be passed through: https://docs.wandb.ai/guides/track/config#efficient-initialization